db/views/analytics_exits_v02.sql

```diff
@@ -0,0 +1,66 @@
+SELECT
+  "ExitID",
+  "EnrollmentID",
+  "PersonalID",
+  "ExitDate",
+  "Destination",
+  "OtherDestination",
+  "AssessmentDisposition",
+  "OtherDisposition",
+  "HousingAssessment",
+  "SubsidyInformation",
+  "ConnectionWithSOAR",
```

Comment:
Suggest removing this, not sure why its on  the Exit table, this is a field on IncomeBenefit (in the current spec)
```suggestion
```